### ISSUE TITLE
Search Checklist: Search updates for Volvo migration (2 of 2)

### ISSUE THREAD

**Issue by mollyaljenkins** [aemsites/volvotrucks-na#50]
Following the establishment of the new Volvo AEM repo, Search needs to be updated so that the new tenant is supported. 

# SharePoint Tasks

- [x] aemsites/vg-volvotrucks-us#744

# Helix Tasks

 aemsites/vg-volvotrucks-us#755
- [x] Add language field on helix-query.yaml | meta property locale
- [x] ask to receive domain name url from helix
- [x] receive sitename in unpublish trigger event (repository_dispatch) (blocked by Adobe Sep25)

**Search Tasks @netmanito:** 

# Task 1 (2 SP)
- [x] aemsites/volvotrucks-na#46

## Subtasks
- [x] New folder Eds-connector
- [x] Modify index-name unique `eds-search` 
- [x] Change website-name to be a value in tenant field * from documents
- [x] Generate URL field with path
- [x] Generate unique UUID based on website-name+path
- [x] remove ingestion processor URL and tenant
- [x] repository_dispatch information needed
- [x] Generate unique UUID based on website-name+path
- [x] Delete based on  UUID

# Task 2 (2 SP)
- [x] aemsites/vg-volvotrucks-us#747
## Subtasks
- [x] Import files from frankling and create new endpoint eds CI/CD
- [x] add new eds-connector to builds CI/CD
 
# Task 3 (3 SP)
- [x] aemsites/volvotrucks-na#45
## Subtasks
- [x]  Create Search Templates for Main Search and Suggests
- [x]  Remove/update set-url processor from pipeline
- [x]  add language filtering in query
- [x]  add tenant in query filtering

# Task 4 (5 SP)
- [x] aemsites/volvotrucks-na#44
## Subtasks
- [x] New GraphQL endpoint (Duplicate from current VolvoSearch) named EdsSearch
- [x] use unique index based on tenant (websitename)
- [x] use tenant parameter on query filter
- [x] new language filter (default EN)
- [x] Fix index.md files handling in connector (branch with fix but maybe rename to use other task related, pending PR for definitive branch name or issue number) https://github.com/aemsites/volvotrucks-na/tree/749-search-task-4-of-4-deploy-the-graphql | @netmanito

# Out of Scope Tasks
- [ ] Updating pages to support the Search for new pages @Kannan-CM 
- [ ] Adjust search to have the AEM repo as the origin @netmanito @manuel-vara 
- [x] aemsites/volvotrucks-na#40
- [x] Check macktrucks helix5 helix-query.yaml for search @manuel-vara @netmanito 
- [ ] Front end tickets aemsites/volvotrucks-na#33 https://github.com/hlxsites/vg-macktrucks-com/issues/808
- [ ] aemsites/volvotrucks-na#25
- [ ] aemsites/volvotrucks-na#24
